admin/src/js/controllers/images-branding.js

```diff
@@ -25,8 +25,16 @@ angular.module('controllers').controller('ImagesBrandingCtrl',
       return DB().get(DOC_ID, { attachments: true })
         .then(doc => {
           $scope.doc = doc;
-          $scope.favicon = doc._attachments[doc.resources.favicon];
-          $scope.icon = doc._attachments[doc.resources.icon];
+          if (!Object.hasOwn($scope.doc, 'title')) {
+            $scope.doc.title = '';
+          }
+          if (!Object.hasOwn($scope.doc, 'resources')) {
+            $scope.doc.resources = {};
+          }
+          if (Object.hasOwn(doc, '_attachments') && Object.hasOwn(doc, 'resources')) {
+            $scope.favicon = doc._attachments[doc.resources.favicon];
```

Comment:
This might still throw an error if `doc.resources` exists but it has a value that cannot have the `favicon` property, for example `null`. 

<img width="801" height="48" alt="image" src="https://github.com/user-attachments/assets/7c91c1cc-d136-4c40-8bf9-892c98e90d50" />

Reply:
I'll check this one, I didn't see this error.

Reply:
Added some `ng-if` attribute in the HTML to prevent this errors

Reply:
This case is now cover, sorry for the delay I was sick for almost a week :sneezing_face: